resource-aggregate/cqrs/eventstore/mongodb/config.go

```diff
@@ -38,8 +38,8 @@ func WithTLS(cfg *tls.Config) Option {
 type Config struct {
 	URI             string        `long:"uri" env:"URI" envconfig:"URI" default:"mongodb://localhost:27017"`
 	DatabaseName    string        `long:"dbName" env:"DATABASE" envconfig:"DATABASE" default:"eventStore"`
-	BatchSize       int           `long:"batchSize" env:"BATCH_SIZE" envconfig:"BATCH_SIZE" default:"16"`
-	MaxPoolSize     uint64        `long:"maxPoolSize" env:"MAX_POOL_SIZE" envconfig:"MAX_POOL_SIZE" default:"16"`
+	BatchSize       int           `long:"batchSize" env:"BATCH_SIZE" envconfig:"BATCH_SIZE" default:"64"`
+	MaxPoolSize     uint64        `long:"maxPoolSize" env:"MAX_POOL_SIZE" envconfig:"MAX_POOL_SIZE" default:"64"`
```

Comment:
Was this change intended? Why you are changing it for 4times more?

Reply:
No it doesn't need to be changed in future  I want to drive default value by num cores.